cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/PredefinedAbstainDRep.hs

```diff
@@ -0,0 +1,334 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.Testnet.Test.LedgerEvents.Gov.PredefinedAbstainDRep
+  ( hprop_check_predefined_abstain_drep
+  ) where
+
+import           Cardano.Api as Api
+import           Cardano.Api.Error (displayError)
+
+import           Cardano.Testnet
+
+import           Prelude
+
+import           Control.Monad (void)
+import           Control.Monad.Catch (MonadCatch)
+import qualified Data.Aeson as Aeson
+import qualified Data.Aeson.Lens as AL
+import           Data.ByteString.Lazy.Char8 (pack)
+import           Data.String (fromString)
+import qualified Data.Text as Text
+import           Data.Word (Word32)
+import           GHC.Stack (callStack)
+import           Lens.Micro ((^?))
+import           System.FilePath ((</>))
+
+import           Testnet.Components.DReps (createCertificatePublicationTxBody, createVotingTxBody,
+                   generateVoteFiles, retrieveTransactionId, signTx, submitTx)
+import           Testnet.Components.Query (EpochStateView, findLargestUtxoForPaymentKey,
+                   getCurrentEpochNo, getEpochStateView, getMinDRepDeposit)
+import           Testnet.Defaults (defaultDRepKeyPair, defaultDelegatorStakeKeyPair)
+import qualified Testnet.Process.Cli as P
+import qualified Testnet.Process.Run as H
+import qualified Testnet.Property.Utils as H
+import           Testnet.Runtime
+
+import           Hedgehog
+import qualified Hedgehog.Extras as H
+import qualified Hedgehog.Extras.Stock.IO.Network.Sprocket as IO
+
+-- | Execute me with:
+-- @DISABLE_RETRIES=1 cabal test cardano-testnet-test --test-options '-p "/Predefined Abstain DRep/"'@
+hprop_check_predefined_abstain_drep :: Property
```

Comment:
This is missing a comment detailing exactly what this property is testing.

Reply:
Fixed here: https://github.com/IntersectMBO/cardano-node/commit/6f2ffed039101b8f0ad53258c7673bfab2294f73